README.md

```diff
@@ -187,6 +187,20 @@ finetuner.encode(model=model, data=da)
 da.summary()
 ```
 
+When encoding, you can provide data either as a DocumentArray or a list. Since the modality of your input data can be inferred from the model being used, there is not need to provide any additional information besides the content you want to encode:
```

Comment:
```suggestion
When encoding, you can provide data either as a DocumentArray or a list. Since the modality of your input data can be inferred from the model being used, there is no need to provide any additional information besides the content you want to encode:
```

Maybe you can put the second sentence in an info / hint box below the code.